addons/website/views/snippets/snippets.xml

```diff
@@ -219,8 +219,9 @@
 
     <div data-js="layout_column"
         data-selector="section"
-        data-target="> * > .row:not(.s_nb_column_fixed)">
+        data-target="> *:has(> .row:not(.s_nb_column_fixed)), > .o_we_allow_columns">
```

Comment:
In fact, the class should not contain `we` since it does not come from web_editor. So either `o_allow_columns` or `s_allow_columns` since it is something specific to snippets (like the `s_nb_column_fixed` here) :+1: